packages/lesswrong/components/posts/TableOfContents/FixedPositionToC.tsx

```diff
@@ -145,17 +189,50 @@ const styles = (theme: ThemeType) => ({
       marginLeft: 4,
     },
   },
+  comments: {
+    paddingBottom: 20,
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    color: theme.palette.link.tocLink,
+    display: "flex",
+    alignItems: "center",
+    marginRight: 20,
+  },
+  commentsIcon: {
+    fontSize: 20,
+    marginRight: 6,
+    color: theme.palette.grey[400]
+  },
+  '@font-face': {
+    fontFamily: "ETBookBold",
+    src: "url('https://res.cloudinary.com/lesswrong-2-0/raw/upload/v1504470690/et-book-bold-line-figures_piiabg.woff') format('woff')",  
```

Comment:
I don't think this is the right way to do fonts.  We normally load them from the urls provided in `externalStyles.ts` (though I'm not sure that works for `woff` files).  Is the idea that we only use this font in one place and don't want to load it otherwise?

Reply:
Maybe @jimrandomh knows?

Reply:
We could write a bembo.css file, upload it to cloudinary, and then use it in the same way as we currently do font imports. I don't super like it because it moves more code outside of the codebase. 

I think this way of doing fonts is fine.